drm-ffi/src/mode.rs

```diff
@@ -536,13 +536,13 @@ pub fn get_property_blob(
         ioctl::mode::get_blob(fd, &mut blob)?;
     }
 
-    map_shrink!(data, blob.length as usize);
+    map_shrink!(&mut data, blob.length as usize);
```

Comment:
Is it necessary to update the fat pointer stored in `data: Option`, when the caller doesn't get to see this? It's not passing `data: &mut Option<&mut [u8]>` (nor `&mut &mut [u8]` anymore).

Reply:
Good point. But the solution here, is probably, that this call should instead accept a `&mut &mut` again to make a smaller size visible to the user, without requiring a second call to figure out the length.

Reply:
Yup, essentially changing it back to the original type/`fn` signature :)

Reply:
Nope, was an unintended mistake. Thanks for noticing!